src/Sharpliner/AzureDevOps/ConditionedExpressions/ConditionExtensions.cs

```diff
@@ -36,6 +36,14 @@ public static Conditioned<VariableBase> Variable(this IfCondition condition, str
     public static Conditioned<VariableBase> Variable(this IfCondition condition, string name, int value)
         => Conditioned.Link<VariableBase>(condition, new Variable(name, value));
 
+    /// <summary>
+    /// Defines a variable.
+    /// </summary>
+    /// <param name="condition">Conditioned definition</param>
+    /// <param name="variable">Variable instance</param>
+    public static Conditioned<VariableBase> Variable(this IfCondition condition, Variable variable)
```

Comment:
One more question/idea - does it make sense to change this to maybe accept the value as a third parameter too? So that we can just do `Variable(DotnetVersion, "6.0.100")`?

Reply:
I think we could have both, as now we could use `Variable(DotnetVersion with { Value = "6.0.100" })`

Reply:
I just tried this, this doesn't work since now the `Variable` has an implicit conversion to `string` and the overload of `Variable(this IfCondition condition, string name, string value)` is used